components/form.js

```diff
@@ -80,31 +104,73 @@ export default class Form extends React.Component {
   componentDidUpdate(prevProps, prevState) {
     localStorage.setItem("Form Fields", JSON.stringify(this.state.formFields));
     localStorage.setItem("Bearer Token", JSON.stringify(this.state.token));
+    window.addEventListener("online", this.isOnlineEvent);
   }
 
   // Handler html For when the submit button is pressed
   onSubmit = e => {
     e.preventDefault();
-    postFormData(this.state.formData, this.state.token.access_token);
+    let formDataArray = [];
+    if (navigator.onLine) {
+      postFormData(this.state.formData, this.state.token.access_token);
```

Comment:
We should show the user _some_ feedback about the request . Can you show the user if the request was successful or not so they know what's happening in the background?